src/helpers/dom.js

```diff
@@ -7,18 +6,18 @@ import { ScrollToPlugin } from "gsap/ScrollToPlugin";
 // eslint-disable-next-line
 const scrollPlugin = ScrollToPlugin;
 
-export const smoothScrollTo = (node, options = {}) => {
-  const { offset = 0, duration = 0.55 } = options;
+export const smoothScrollTo = (node, onCompleteFn) => {
+  const offset = 0;
+  const duration = 0.55;
 
   TweenLite.to(window, duration, {
     scrollTo: {
       y: node.offsetTop + offset,
     },
+    onComplete: onCompleteFn,
```

Comment:
onCompleteFn is where we set `isNavClicked` to false once the smooth scroll behavior is completed.